src/lava/magma/core/model/py/model.py

```diff
@@ -104,9 +104,7 @@ def lrn_guard(self):
     def post_guard(self):
         pass
 
-    def host_guard(self):
-        pass
-
+    # TODO: (PP) need to handle PAUSE command
```

Comment:
Please file issue. Pause should always allow system to proceed to last phase of time step and then to into HOST phase.

Reply:
Issue #53 filed